static/js/apps/explore/app.tsx

```diff
@@ -269,6 +269,7 @@ export function App(props: { isDemo: boolean }): JSX.Element {
     const topic = getSingleParam(hashParams[URL_HASH_PARAMS.TOPIC]);
     const place = getSingleParam(hashParams[URL_HASH_PARAMS.PLACE]);
     const dc = getSingleParam(hashParams[URL_HASH_PARAMS.DC]);
+    const idx = getSingleParam(hashParams[URL_HASH_PARAMS.IDX]);
```

Comment:
what is this idx?

Reply:
This is the embeddings index listed in embeddings.yaml.  I added a comment.

Reply:
Mainly for testing.